docs/search-views/semantic-search/tasks/create-semantic-search-views.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 10
+title: Create a Semantic Search View
+---
+
+This page explains how to create a new semantic search view.
+
+## Prerequisites
+
+- A [Macrometa account](https://auth-play.macrometa.io/) with sufficient permissions to create search views.
+- At least one collection created.
+
+## Create a Semantic Search View
+
+Follow these instructions to create a new semantic search view using the GDN console web UI.
+
+1. [Log in to your Macrometa account](https://auth-play.macrometa.io/).
+2. Click **Data > Search Views**.
+3. Click **New View**.
+4. Enter a **Search View Name**.
+5. In **Collection**, select a collection.
+6. In **Field**, enter a field to be indexed in the search view.
```

Comment:
The field must correspond to an array of numbers in the documents in the collection, and the vector size (next field) must be the size of the array. e.g. if your document is {"_key": "100", "name": "n1", "address": "a1", "vector_name": [30, 40,50, 60]}, and you want to search for similar names based on vector_name, use "vector_name" as the field and 4 as vector size.